package.json

```diff
@@ -5,25 +5,22 @@
   "repository": "https://github.com/tl-its-umich-edu/my-learning-analytics",
   "license": "Apache-2.0",
   "dependencies": {
-    "@apollo/react-hooks": "^3.1.1",
-    "@fortawesome/fontawesome-free": "5.11.2",
-    "@fortawesome/react-fontawesome": "0.1.8",
-    "@fortawesome/fontawesome-svg-core": "1.2.32",
-    "apollo-boost": "^0.4.4",
+    "@apollo/client": "^3.5.10",
+    "@fortawesome/fontawesome-free": "6.1.1",
+    "@fortawesome/fontawesome-svg-core": "6.1.1",
+    "@fortawesome/react-fontawesome": "0.1.18",
     "clsx": "^1.1.1",
-    "d3": "^5.12.0",
+    "d3": "^5.16.0",
     "d3-tip": "~0.9.1",
```

Comment:
Upgrading `d3` is a significant piece still to be done, and which I think we'll tackle separately. We're behind two major releases and the `d3-tip` companion library seems to have been archived on GitHub (https://github.com/caged/d3-tip). I found a migration guide for version 6, but no equivalent yet for version 7: https://observablehq.com/@d3/d3v6-migration-guide

Reply:
We would want to upgrade to latest since 2 of our visuals are using d3 solution. But if this not a huge risk we can handle this separately. I don't know if this is a minor, patch Myla release.

Reply:
I'd say this PR has enough changes where adding `d3` stuff would put it overboard (if isn't already). We can discuss as a team whether `d3` needs to be updated in this release, but again, I don't see any vulnerabilities right now for this